snowpack.config.mjs

```diff
@@ -0,0 +1,41 @@
+/** @type {import("snowpack").SnowpackUserConfig } */
```

Comment:
Please add the source of this file/indicate if its auto-generated.

Reply:
This is snowpack configuration file, its mostly been generated by the initial bootstrap -- with a couple added configurations

Reply:
Ok. Please add that as a comment and also add comments on the ones you added (and why). 
Imo its extremely important to keep config files well documented

Reply:
Okay, I added info about snowpack's optimize option which the app using right now for builds. They recommend the Webpack plugin for prod but their built-in bundler/minimizer seems to be doing a good job right now. And I also added that this is the Snowpack config file at the top.

The other options I set already had comments on them:

```ts
    buildOptions: {
        // Enable source maps for coverage
        sourcemap: true,
        // put the meta snowpack build files under snowpack instead of _snowpack since Github special-cases underscore prefixed folders
        metaUrlPath: 'snowpack'
    }
```